client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/composables/table/BpTable.kt

```diff
@@ -115,4 +101,95 @@ fun <T> ColumnScope.BpTable(
             }
         }
     }
-}
\ No newline at end of file
+}
+
+@Composable
+private fun TableRowLoading(
```

Comment:
The TableRowLoading can be a part of The TableRow composable and change the content of row depend on the loading state if possible

Reply:
i think number of row depend on data so if i make condition in Table Row the loading show in single row but i need make him depend on limit that in each page